public/Day 100/index.html

```
@@ -5,6 +5,8 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
   <title>SERVER: Survival Protocol</title>
+  <meta name="description"
```

HTMLHint (1.9.1)

[warning] The empty tag : [ meta ] must be self closed.

(tag-self-close)